### PR TITLE
chore: format code with google java style guide

### PR DESCRIPTION
**Pull request by duanyytop**
* Format all codes with Google Java Style Guide

### CODE REVIEW

**Comment by duanyytop** [comment]
> Question: when other developers work on this, will this guide/lint apply and show warning/errors when they don't flow the style?

Two actions:
1. We can add lint of Google Java Style Code, so if his code doesn't flow the style , he will not commit his code. It will show warning/errors when they don't flow the style.
2. Add configuration of Google Java Style Code in IntelliJ IDEA IDE when saving code. It will not show warning/errors when they don't flow the style. But we can give suggestion in readme and point out when reviewing code from other developers.

Now I use the 2 action, considering that we just have two developers. And I will import action 1 later.

**Comment by ashchan** [comment]
Then please hold on this. Update files fixing format when when bring lint tool.

**Comment by duanyytop** [comment]
Add Google Java Code Style Guide gradle plugin and Google Checkstyle to promise other developers' code to follow our code style.

If other developers' code doesn't follow our style, he will be blocked by format error when he commits code, and I write some solutions for fixing format errors in readme.

Now I think we can promise other developers to follow our code style

File: README.md [hunk 1]
```diff
@@ -10,3 +10,12 @@ git clone https://github.com/nervosnetwork/ckb-sdk-java.git
...
 You will get `console-{version}-all.jar` from console module. 
+
+### How to Develop
+
+We use [Google Java Code Format](https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping) and [google checkstyle](https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml) to develop.
+
+If `verifyGoogleJavaFormat FAILED` happens when you build this project, please reformat your code with [Google Java Code Format](https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping) 
+or execute `./gradlew goJF` in mac and linux or `gradlew goJF` in windows.
+
+If you use IntelliJ IDEA to develop, you can install `google-java-format` plugin to reformat code automatically.
```
Comment (ashchan) [inline README.md]:
```suggestion
If you use IntelliJ IDEA to develop, you can install `google-java-format` plugin to format code automatically.
```

File: README.md [hunk 2]
```diff
@@ -10,3 +10,12 @@ git clone https://github.com/nervosnetwork/ckb-sdk-java.git
 gradle shadowJar  // ./gradlew shadowJar 
 ```
 You will get `console-{version}-all.jar` from console module. 
+
+### How to Develop
+
+We use [Google Java Code Format](https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping) and [google checkstyle](https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml) to develop.
+
+If `verifyGoogleJavaFormat FAILED` happens when you build this project, please reformat your code with [Google Java Code Format](https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping)
```
Comment (ashchan) [inline README.md]:
```suggestion
If `verifyGoogleJavaFormat FAILED` happens when you build this project, please format your code with [Google Java Code Format](https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping) 
```

File: README.md [hunk 3]
```diff
@@ -10,3 +10,12 @@ git clone https://github.com/nervosnetwork/ckb-sdk-java.git
 gradle shadowJar  // ./gradlew shadowJar 
 ```
 You will get `console-{version}-all.jar` from console module. 
+
+### How to Develop
+
+We use [Google Java Code Format](https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping) and [google checkstyle](https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml) to develop.
+
+If `verifyGoogleJavaFormat FAILED` happens when you build this project, please reformat your code with [Google Java Code Format](https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping) 
+or execute `./gradlew goJF` in mac and linux or `gradlew goJF` in windows.
```
Comment (ashchan) [inline README.md]:
```suggestion
or execute `./gradlew goJF` on macOS and Linux,  or `gradlew goJF` on Windows.
```

**Pull request merged**